src/jit/gentree.cpp

```diff
@@ -304,7 +304,8 @@ void GenTree::InitNodeSize()
     GenTree::s_gtNodeSizes[GT_INDEX_ADDR]        = TREE_NODE_SZ_LARGE;
     GenTree::s_gtNodeSizes[GT_ARR_BOUNDS_CHECK] = TREE_NODE_SZ_LARGE;
 #ifdef FEATURE_SIMD
-    GenTree::s_gtNodeSizes[GT_SIMD_CHK] = TREE_NODE_SZ_LARGE;
+    GenTree::s_gtNodeSizes[GT_SIMD_CHK]         = TREE_NODE_SZ_LARGE;
+    GenTree::s_gtNodeSizes[GT_SIMD]             = TREE_NODE_SZ_LARGE;
```

Comment:
Why is this change needed? We shouldn't be touching the GT_SIMD code.

Reply:
I have not touched it - clang -format and clang-tidy did it by creating format patch

Reply:
Hmmm.... I wouldn't expect clang format or clang-tidy to add new code, only move around existing code.

Reply:
Yeah, it's my copy paste error around place flagged by clang-format